contracts/TokenHub.sol

```diff
@@ -522,6 +522,19 @@ contract TokenHub is ITokenHub, System, IParamSubscriber, IApplication, ISystemR
     return elements.encodeList();
   }
 
+  /**
+   * @dev request a BC token aridrop transfer from BSC
+   *
+   * @param contractAddr The token contract which is transferred
+   * @param recipient The destination address of the transfer on BSC.
+   * @param amount The amount to transfer
+   */
+  function unlock(address contractAddr, address recipient, uint256 amount) external override onlyInit payable {
+    require(IBEP20(contractAddr).balanceOf(address(this)) < amount, "InsufficientBalance");
```

Comment:
Should be `IBEP20(contractAddr).balanceOf(address(this)) > amount`?

Reply:
you are right

Reply:
fixed